### PR TITLE
Added simple edit todo functionality

### PR DESCRIPTION
**Pull request by ychan052**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -58,6 +58,16 @@ function App() {
       setTodos([...todos, newTodo]);
     }
   };
+  const editTodo = (id) => {
+    let text = window.prompt('Edit Todo', '');
```
Comment (max-programming) [inline src/App.js:62]: I'll merge this for now but it is not good to use merge. Maybe you can create a modal

**Comment by max-programming** [comment]
Very very important feature! Thanks a lot Nicholas!!! 👏

File: src/App.js (hunk 1 above)
Comment (ychan052) [reply src/App.js:62]: I will work on that!

Comment (max-programming) [reply src/App.js:62]: Make a Pull so that you keep up with the latest code. Thanks for the edit functionality

**Pull request merged**